### DISCUSSION TITLE
zoom connectivity

### DISCUSSION THREAD

**Discussion by shehrozayub800** [AammarTufail/python_ka_chilla2023#15] [Q&A]
zoom me daily kese connect kren gy? muje nahi aata.

**Comment by shehrozayub800** [comment]
pin waly link sy open nahi hua

**Reply by AammarTufail** [reply]
Read pinned msg of telegram

**Comment by shehrozayub800** [comment]
Sir isi link sy open kiya tha nahi hua. Ap koi save krne ka tareka bta den
k wo save ho jay

**Reply by SahilJatoi744** [reply]
Save the link in a Word file and then directly open it from over there

**Comment by Masoodullah909** [comment]
> zoom me daily kese connect kren gy? muje nahi aata.

link in the pinned message of a Telegram group
